CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The version headers in this history reflect the versions of Apollo Server itself
 ## vNEXT
 
 - Add a few missing dependencies to packages. [PR #6393](https://github.com/apollographql/apollo-server/pull/6393)
+- Add `embed` as an option in the `ApolloServerPluginLandingPageLocalDefault` and `ApolloServerPluginLandingPageProductionDefault` plugins. In `ApolloServerPluginLandingPageLocalDefault`, when `embed: true` is passed, the Apollo Studio Sandbox will be embedded on your Apollo Server endpoint. In `ApolloServerPluginLandingPageProductionDefault`, when `embed` is `true`, or is a config object, the Apollo Studio embedded Explorer will be embedded on your Apollo Server endpoint. In both cases, users are able to query against their endpoint from their endpoint without whitelisting Apollo urls for CORS.
```

Comment:
last sentence maybe "users can use the embedded app to run GraphQL operations without any special CORS setup"?